packages/fxa-payments-server/src/routes/Subscriptions/index.tsx

```diff
@@ -136,32 +137,36 @@ export const Subscriptions = ({
     return <LoadingOverlay isLoading={true} />;
   }
 
-  if (profile.error !== null) {
+  if (!profile.result || profile.error !== null) {
```

Comment:
Ensuring that these fetched results are both not errors and have data loaded helps simplify some null guards later on. Not directly relevant to this PR's issue, but helped with the refactoring a bit.